game/strategy.py

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python
+from abc import ABCMeta, abstractmethod
+
+class Strategy(object):
+    __metaclass__ = ABCMeta
+
+    @abstractmethod
+    def return_column(self, board):
```

Comment:
Not something to address now, but maybe you can find a better name for this message